application/src/main/java/org/opentripplanner/apis/transmodel/TransmodelGraphQLSchema.java

```diff
@@ -683,24 +687,18 @@ private GraphQLSchema create() {
                 environment.getArgument("maximumLatitude")
               )
             );
-            return GqlUtil
-              .getTransitService(environment)
-              .findRegularStops(envelope)
-              .stream()
-              .filter(stop -> envelope.contains(stop.getCoordinate().asJtsCoordinate()))
-              .filter(stop ->
-                environment.getArgument("authority") == null ||
-                stop.getId().getFeedId().equalsIgnoreCase(environment.getArgument("authority"))
-              )
-              .filter(stop -> {
-                boolean filterByInUse = TRUE.equals(environment.getArgument("filterByInUse"));
-                boolean inUse = !GqlUtil
-                  .getTransitService(environment)
-                  .findPatterns(stop, true)
-                  .isEmpty();
-                return !filterByInUse || inUse;
-              })
-              .collect(Collectors.toList());
+
+            var authority = environment.<String>getArgument("authority");
+            var filterInUse = environment.<Boolean>getArgument("filterByInUse");
+
+            RegularStopRequest regularStopRequest = RegularStopRequest
+              .of()
+              .withEnvelope(envelope)
+              .withFeed(authority)
```

Comment:
This is highly dubious. I don't think that a feed is the same as an agency.

Reply:
Ah I see that it was like that before. Still I don't think its correct.

Reply:
Changed it to be agency.